packages/bpk-component-datepicker/src/BpkDatepicker-test.js

```diff
@@ -204,6 +204,34 @@ describe('BpkDatepicker', () => {
     expect(datepicker.state('isOpen')).toEqual(true);
   });
 
+  it('should open when the isOpen prop is changed from the outside', () => {
```

Comment:
Maybe a test for we should not change isOpen when `prevProps.isOpen === isOpen` and `props.isOpen !== state.isOpen`?